tests/UnitTests/CoreGraphics.drawing/CGContextDrawing_ClearRectTests.cpp

```diff
@@ -0,0 +1,102 @@
+        return CFStringCreateWithFormat(nullptr,
+                                        nullptr,
+                                        CFSTR("TestImage.CGContext.ClearRect.rect.(%0.0f.%0.0f)%0.0fx%0.0f.transformationAffine.[%0.1f,%0."
+                                              "1f,%0.1f,%0.1f,%0.1f,%0.1f].png"),
+                                        rect.origin.x,
+                                        rect.origin.y,
+                                        rect.size.width,
+                                        rect.size.height,
+                                        transformation.a,
+                                        transformation.b,
+                                        transformation.c,
+                                        transformation.d,
+                                        transformation.tx,
+                                        transformation.ty);
+    }
+};
+
+TEST_P(CGClearRect, Transformed) {
+    CGRect rect = ::testing::get<0>(GetParam());
+    CGAffineTransform transformation = ::testing::get<1>(GetParam());
+
+    CGContextRef context = GetDrawingContext();
+
+    CGContextAddRect(context, CGRectMake(100, 50, 50, 50));
+    CGContextConcatCTM(context, transformation);
+    CGContextClip(context);
+
+    CGContextSetRGBFillColor(context, 0.48, 0.73, 0, 1);
+    CGContextClearRect(context, rect);
+}
+
+TEST_P(CGClearRect, Transformed2) {
+    CGRect rect = ::testing::get<0>(GetParam());
+    CGAffineTransform transformation = ::testing::get<1>(GetParam());
+
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextAddRect(context, CGRectMake(0, 0, 250, 250));
+    CGContextConcatCTM(context, transformation);
+    CGContextClip(context);
+
+    CGContextSetRGBFillColor(context, 0.48, 0.73, 0, 1);
+    CGContextClearRect(context, rect);
+}
+
+static CGRect rects[] = { CGRectMake(0, 0, 100, 100), CGRectMake(0, 0, 50, 250), CGRectMake(100, 100, 125, 250) };
+static CGAffineTransform transformation[] = { CGAffineTransformMakeRotation(0.4),
+                                              CGAffineTransformMakeTranslation(3, 6),
+                                              CGAffineTransformMakeScale(0.25, 0.75),
+                                              CGAffineTransformMake(1.f, 0.f, 0.3f, 1.f, 0.f, 0.f) };
+
+INSTANTIATE_TEST_CASE_P(CGContextRectClearTests,
+                        CGClearRect,
+                        ::testing::Combine(::testing::ValuesIn(rects), ::testing::ValuesIn(transformation)));
+
+DRAW_TEST_F(CGContextRectClear, ClearRect, WhiteBackgroundTest<>) {
+    CGContextRef context = GetDrawingContext();
+    CGRect bounds = GetDrawingBounds();
+
+    CGContextSetRGBFillColor(context, 1.0, 0.0, 0.0, 1);
+    CGContextFillRect(context, bounds);
+
+    CGRect borderRect = CGRectInset(bounds, 30, 50);
+    CGContextClearRect(context, borderRect);
+}
+
+DISABLED_DRAW_TEST_F(CGContextRectClear, ClearRectRam, WhiteBackgroundTest<>) {
```

Comment:
Disabled?

This one is named `Ram`; perhaps `Arc` is better!

Also, why is it disabled?